docs/intro/tutorial07.txt

```diff
@@ -418,5 +418,39 @@ uses a template variable called ``app_list``. That variable contains every
 installed Django app. Instead of using that, you can hard-code links to
 object-specific admin pages in whatever way you think is best.
 
+How to completely delete an app from the project
+================================================
+Now that you have learned every aspect of app creation, customization and etc.
+You need to know how to properly delete an app from the project.
+
+First, you need to remove all the tables from your database or any other traces 
+the app might have. For this, use:
+.. console::
+
+   $ python manage.py migrate app_name zero
+This command also executes your SQL too.
+
+However, be careful when running this command when you have other apps 
+dependent on the migrations of the deleting app. After it is deleted, 
+migration system will crash because it can not find those migrations.
```

Comment:
Good advice!

Could we suggest do a "squash migrations" first, to avoid this situation? https://docs.djangoproject.com/en/4.1/topics/migrations/#migration-squashing

Reply:
alright, will do